src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random random = new Random();
```

Comment:
```suggestion
    private static final Random random = new Random();
    
    public Colors getRandomColor() {
```
let's make this a global field.
This way Random instance would be created only once, instead of each time you invoke this method